content/news/030/index.md

```diff
@@ -70,6 +70,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Engine Updates
 
+### [Rusty Engine 4.0]
+
+![An example Rusty Engine game](rusty_engine4.png)
```

Comment:
> +623 KB 

The image is too big - could you please reduce its size at least to 300kb?

Reply:
Shoot, I forgot to check. Yes, I'll fix it right up. 👍🏻

Reply:
✅ Done!